Traceback (most recent call last):
  File "tools/test.py", line 367, in <module>
    main()
  File "tools/test.py", line 335, in main
    dataset = build_dataset(cfg.data.test, dict(test_mode=True))
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmaction/datasets/builder.py", line 39, in build_dataset
    dataset = build_from_cfg(cfg, DATASETS, default_args)
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmcv/utils/registry.py", line 44, in build_from_cfg
    raise KeyError(
KeyError: 'EpicKitchensMMSADA is not in the dataset registry'
Traceback (most recent call last):
  File "tools/test.py", line 367, in <module>
    main()
  File "tools/test.py", line 335, in main
    dataset = build_dataset(cfg.data.test, dict(test_mode=True))
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmaction/datasets/builder.py", line 39, in build_dataset
    dataset = build_from_cfg(cfg, DATASETS, default_args)
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmcv/utils/registry.py", line 44, in build_from_cfg
    raise KeyError(
KeyError: 'EpicKitchensMMSADA is not in the dataset registry'
Traceback (most recent call last):
  File "tools/test.py", line 367, in <module>
    main()
  File "tools/test.py", line 335, in main
    dataset = build_dataset(cfg.data.test, dict(test_mode=True))
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmaction/datasets/builder.py", line 39, in build_dataset
    dataset = build_from_cfg(cfg, DATASETS, default_args)
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmcv/utils/registry.py", line 44, in build_from_cfg
    raise KeyError(
KeyError: 'EpicKitchensMMSADA is not in the dataset registry'
Traceback (most recent call last):
  File "tools/test.py", line 367, in <module>
    main()
  File "tools/test.py", line 335, in main
    dataset = build_dataset(cfg.data.test, dict(test_mode=True))
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmaction/datasets/builder.py", line 39, in build_dataset
    dataset = build_from_cfg(cfg, DATASETS, default_args)
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmcv/utils/registry.py", line 44, in build_from_cfg
    raise KeyError(
KeyError: 'EpicKitchensMMSADA is not in the dataset registry'
Traceback (most recent call last):
  File "tools/test.py", line 367, in <module>
    main()
  File "tools/test.py", line 335, in main
    dataset = build_dataset(cfg.data.test, dict(test_mode=True))
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmaction/datasets/builder.py", line 39, in build_dataset
    dataset = build_from_cfg(cfg, DATASETS, default_args)
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmcv/utils/registry.py", line 44, in build_from_cfg
    raise KeyError(
KeyError: 'EpicKitchensMMSADA is not in the dataset registry'
Traceback (most recent call last):
  File "tools/test.py", line 367, in <module>
    main()
  File "tools/test.py", line 335, in main
    dataset = build_dataset(cfg.data.test, dict(test_mode=True))
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmaction/datasets/builder.py", line 39, in build_dataset
    dataset = build_from_cfg(cfg, DATASETS, default_args)
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmcv/utils/registry.py", line 44, in build_from_cfg
    raise KeyError(
KeyError: 'EpicKitchensMMSADA is not in the dataset registry'
Traceback (most recent call last):
  File "tools/test.py", line 367, in <module>
    main()
  File "tools/test.py", line 335, in main
    dataset = build_dataset(cfg.data.test, dict(test_mode=True))
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmaction/datasets/builder.py", line 39, in build_dataset
    dataset = build_from_cfg(cfg, DATASETS, default_args)
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmcv/utils/registry.py", line 44, in build_from_cfg
    raise KeyError(
KeyError: 'EpicKitchensMMSADA is not in the dataset registry'
Traceback (most recent call last):
  File "tools/test.py", line 367, in <module>
    main()
  File "tools/test.py", line 335, in main
    dataset = build_dataset(cfg.data.test, dict(test_mode=True))
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmaction/datasets/builder.py", line 39, in build_dataset
    dataset = build_from_cfg(cfg, DATASETS, default_args)
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmcv/utils/registry.py", line 44, in build_from_cfg
    raise KeyError(
KeyError: 'EpicKitchensMMSADA is not in the dataset registry'
Traceback (most recent call last):
  File "tools/test.py", line 367, in <module>
    main()
  File "tools/test.py", line 335, in main
    dataset = build_dataset(cfg.data.test, dict(test_mode=True))
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmaction/datasets/builder.py", line 39, in build_dataset
    dataset = build_from_cfg(cfg, DATASETS, default_args)
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmcv/utils/registry.py", line 44, in build_from_cfg
    raise KeyError(
KeyError: 'EpicKitchensMMSADA is not in the dataset registry'
Traceback (most recent call last):
  File "tools/test.py", line 367, in <module>
    main()
  File "tools/test.py", line 335, in main
    dataset = build_dataset(cfg.data.test, dict(test_mode=True))
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmaction/datasets/builder.py", line 39, in build_dataset
    dataset = build_from_cfg(cfg, DATASETS, default_args)
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmcv/utils/registry.py", line 44, in build_from_cfg
    raise KeyError(
KeyError: 'EpicKitchensMMSADA is not in the dataset registry'
Traceback (most recent call last):
  File "tools/test.py", line 367, in <module>
    main()
  File "tools/test.py", line 335, in main
    dataset = build_dataset(cfg.data.test, dict(test_mode=True))
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmaction/datasets/builder.py", line 39, in build_dataset
    dataset = build_from_cfg(cfg, DATASETS, default_args)
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmcv/utils/registry.py", line 44, in build_from_cfg
    raise KeyError(
KeyError: 'EpicKitchensMMSADA is not in the dataset registry'
Traceback (most recent call last):
  File "tools/test.py", line 367, in <module>
    main()
  File "tools/test.py", line 335, in main
    dataset = build_dataset(cfg.data.test, dict(test_mode=True))
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmaction/datasets/builder.py", line 39, in build_dataset
    dataset = build_from_cfg(cfg, DATASETS, default_args)
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmcv/utils/registry.py", line 44, in build_from_cfg
    raise KeyError(
KeyError: 'EpicKitchensMMSADA is not in the dataset registry'
Traceback (most recent call last):
  File "tools/test.py", line 367, in <module>
    main()
  File "tools/test.py", line 335, in main
    dataset = build_dataset(cfg.data.test, dict(test_mode=True))
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmaction/datasets/builder.py", line 39, in build_dataset
    dataset = build_from_cfg(cfg, DATASETS, default_args)
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmcv/utils/registry.py", line 44, in build_from_cfg
    raise KeyError(
KeyError: 'EpicKitchensMMSADA is not in the dataset registry'
Traceback (most recent call last):
  File "tools/test.py", line 367, in <module>
    main()
  File "tools/test.py", line 335, in main
    dataset = build_dataset(cfg.data.test, dict(test_mode=True))
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmaction/datasets/builder.py", line 39, in build_dataset
    dataset = build_from_cfg(cfg, DATASETS, default_args)
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmcv/utils/registry.py", line 44, in build_from_cfg
    raise KeyError(
KeyError: 'EpicKitchensMMSADA is not in the dataset registry'
Traceback (most recent call last):
  File "tools/test.py", line 367, in <module>
    main()
  File "tools/test.py", line 335, in main
    dataset = build_dataset(cfg.data.test, dict(test_mode=True))
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmaction/datasets/builder.py", line 39, in build_dataset
    dataset = build_from_cfg(cfg, DATASETS, default_args)
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmcv/utils/registry.py", line 44, in build_from_cfg
    raise KeyError(
KeyError: 'EpicKitchensMMSADA is not in the dataset registry'
Traceback (most recent call last):
  File "tools/test.py", line 367, in <module>
    main()
  File "tools/test.py", line 335, in main
    dataset = build_dataset(cfg.data.test, dict(test_mode=True))
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmaction/datasets/builder.py", line 39, in build_dataset
    dataset = build_from_cfg(cfg, DATASETS, default_args)
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmcv/utils/registry.py", line 44, in build_from_cfg
    raise KeyError(
KeyError: 'EpicKitchensMMSADA is not in the dataset registry'
Traceback (most recent call last):
  File "tools/test.py", line 367, in <module>
    main()
  File "tools/test.py", line 335, in main
    dataset = build_dataset(cfg.data.test, dict(test_mode=True))
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmaction/datasets/builder.py", line 39, in build_dataset
    dataset = build_from_cfg(cfg, DATASETS, default_args)
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmcv/utils/registry.py", line 44, in build_from_cfg
    raise KeyError(
KeyError: 'EpicKitchensMMSADA is not in the dataset registry'
Traceback (most recent call last):
  File "tools/test.py", line 367, in <module>
    main()
  File "tools/test.py", line 335, in main
    dataset = build_dataset(cfg.data.test, dict(test_mode=True))
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmaction/datasets/builder.py", line 39, in build_dataset
    dataset = build_from_cfg(cfg, DATASETS, default_args)
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmcv/utils/registry.py", line 44, in build_from_cfg
    raise KeyError(
KeyError: 'EpicKitchensMMSADA is not in the dataset registry'
Traceback (most recent call last):
  File "tools/test.py", line 367, in <module>
    main()
  File "tools/test.py", line 335, in main
    dataset = build_dataset(cfg.data.test, dict(test_mode=True))
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmaction/datasets/builder.py", line 39, in build_dataset
    dataset = build_from_cfg(cfg, DATASETS, default_args)
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmcv/utils/registry.py", line 44, in build_from_cfg
    raise KeyError(
KeyError: 'EpicKitchensMMSADA is not in the dataset registry'
Traceback (most recent call last):
  File "tools/test.py", line 367, in <module>
    main()
  File "tools/test.py", line 335, in main
    dataset = build_dataset(cfg.data.test, dict(test_mode=True))
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmaction/datasets/builder.py", line 39, in build_dataset
    dataset = build_from_cfg(cfg, DATASETS, default_args)
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmcv/utils/registry.py", line 44, in build_from_cfg
    raise KeyError(
KeyError: 'EpicKitchensMMSADA is not in the dataset registry'
Traceback (most recent call last):
  File "tools/test.py", line 367, in <module>
    main()
  File "tools/test.py", line 335, in main
    dataset = build_dataset(cfg.data.test, dict(test_mode=True))
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmaction/datasets/builder.py", line 39, in build_dataset
    dataset = build_from_cfg(cfg, DATASETS, default_args)
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmcv/utils/registry.py", line 44, in build_from_cfg
    raise KeyError(
KeyError: 'EpicKitchensMMSADA is not in the dataset registry'
Traceback (most recent call last):
  File "tools/test.py", line 367, in <module>
    main()
  File "tools/test.py", line 335, in main
    dataset = build_dataset(cfg.data.test, dict(test_mode=True))
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmaction/datasets/builder.py", line 39, in build_dataset
    dataset = build_from_cfg(cfg, DATASETS, default_args)
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmcv/utils/registry.py", line 44, in build_from_cfg
    raise KeyError(
KeyError: 'EpicKitchensMMSADA is not in the dataset registry'
Traceback (most recent call last):
  File "tools/test.py", line 367, in <module>
    main()
  File "tools/test.py", line 335, in main
    dataset = build_dataset(cfg.data.test, dict(test_mode=True))
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmaction/datasets/builder.py", line 39, in build_dataset
    dataset = build_from_cfg(cfg, DATASETS, default_args)
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmcv/utils/registry.py", line 44, in build_from_cfg
    raise KeyError(
KeyError: 'EpicKitchensMMSADA is not in the dataset registry'
Traceback (most recent call last):
  File "tools/test.py", line 367, in <module>
    main()
  File "tools/test.py", line 335, in main
    dataset = build_dataset(cfg.data.test, dict(test_mode=True))
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmaction/datasets/builder.py", line 39, in build_dataset
    dataset = build_from_cfg(cfg, DATASETS, default_args)
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmcv/utils/registry.py", line 44, in build_from_cfg
    raise KeyError(
KeyError: 'EpicKitchensMMSADA is not in the dataset registry'
Traceback (most recent call last):
  File "tools/test.py", line 367, in <module>
    main()
  File "tools/test.py", line 335, in main
    dataset = build_dataset(cfg.data.test, dict(test_mode=True))
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmaction/datasets/builder.py", line 39, in build_dataset
    dataset = build_from_cfg(cfg, DATASETS, default_args)
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmcv/utils/registry.py", line 44, in build_from_cfg
    raise KeyError(
KeyError: 'EpicKitchensMMSADA is not in the dataset registry'
Traceback (most recent call last):
  File "tools/test.py", line 367, in <module>
    main()
  File "tools/test.py", line 335, in main
    dataset = build_dataset(cfg.data.test, dict(test_mode=True))
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmaction/datasets/builder.py", line 39, in build_dataset
    dataset = build_from_cfg(cfg, DATASETS, default_args)
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmcv/utils/registry.py", line 44, in build_from_cfg
    raise KeyError(
KeyError: 'EpicKitchensMMSADA is not in the dataset registry'
Traceback (most recent call last):
  File "tools/test.py", line 367, in <module>
    main()
  File "tools/test.py", line 335, in main
    dataset = build_dataset(cfg.data.test, dict(test_mode=True))
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmaction/datasets/builder.py", line 39, in build_dataset
    dataset = build_from_cfg(cfg, DATASETS, default_args)
  File "/data/shinpaul14/anaconda3/envs/action-dg/lib/python3.8/site-packages/mmcv/utils/registry.py", line 44, in build_from_cfg
    raise KeyError(
KeyError: 'EpicKitchensMMSADA is not in the dataset registry'
